### PR TITLE
Add NMEA GST Message for GPSInformation.Accuracies

### PR DESCRIPTION
**Pull request by elpaso**
Fix style issues form
https://github.com/qgis/QGIS/pull/9874

### CODE REVIEW

**Comment by nyalldawson** [comment]
From #9874 , upstream has shown no interest in the change since May (see https://sourceforge.net/p/nmea/bugs/18/). Consequently I think we should merge this when ready and depart further from upstream.

**Comment by nyalldawson** [comment]
Also, this says it all really:

image

**Comment by jiargei** [comment]
I don't understand? Do I have to do something?

**Comment by nyalldawson** [comment]
@jiargei No - @elpaso has just adopted your work to try to resolve the CI issues and get it merged.

**Comment by jiargei** [comment]
Ok, nice

Nyall Dawson <notifications@github.com> schrieb am Do., 20. Juni 2019,
04:57:

File: external/nmea/info.h [hunk 1]
```diff
@@ -116,6 +116,18 @@ typedef struct _nmeaINFO
   double  direction;  //!< Track angle in degrees True
   double  declination; //!< Magnetic variation degrees (Easterly var. subtracts from true course)
 
+  double  rms_pr;     //!< RMS value of the pseudorange residuals;
+
+  /**
+   * < Includes carrier phase residuals during periods of RTK (float) and RTK (fixed) processing
```
Comment (m-kuhn) [inline external/nmea/info.h]: Is this docstring for `satinfo`? If yes, the `<` prefix should be removed, the blank line 130 removed and the docstring on 131 removed.

**Comment by jiargei** [comment]
Hej!
I just updated the output format and the tooltip ... can you add this too?

**Comment by elpaso** [comment]
> Hej!
> I just updated the output format and the tooltip ... can you add this too?

where is that commit?

File: external/nmea/info.h (hunk 1 above)
Comment (elpaso) [reply external/nmea/info.h]: I think astyle messed up something, I'll check it

**Comment by elpaso** [comment]
@jiargei merged, thanks! If you make any other change, please open a new PR.

**Pull request merged**